### ISSUE TITLE
Notification settings: no text displayed - only radio buttons

### ISSUE THREAD

**Issue by stefonarch** [lxde/lxqt#744]
It's some time like that, at least with locale it_IT and en_US.utf8 as seen on IRC

![schermata](https://cloud.githubusercontent.com/assets/10681413/8634209/5c016cb4-27f0-11e5-885b-add1dd8928b5.png)

**Comment by jleclanche** [comment]
If there's actually no text this is a pretty nasty design issue.

**Comment by palinek** [comment]
This was intentionaly implemented in lxde/lxqt-panel@41a22ef on request in lxde/lxqt#609

**Comment by jleclanche** [comment]
It's bad design. I should have caught it sooner, sorry.

**Comment by palinek** [comment]
Then... how should it look like?
This is how it was before that "text removal" [commit](https://github.com/lxde/lxqt-notificationd/commit/41a22ef)
![screenshot](https://cloud.githubusercontent.com/assets/10990929/8715551/f474c298-2b83-11e5-8619-8c569c6de233.png)

**Comment by jleclanche** [comment]
A dropdown with topright/topleft/bottomright/bottomleft would be better than both. Could also go fancy and do a whole monitor layout but I don't think that's actually useful.

Also, does anyone actually use the centered notifications? Horizontal center maybe?

**Comment by Vladimir-csp** [comment]
What is wrong with textless design? It is pretty obvious if enclosed in the border.
Also I would vote against dropdowns, these are better for long dynamic lists.

**Comment by intialonso** [comment]
As it is right now it is not obvious.
As @Vladimir-csp says the textless design could be more intuitive if enclosed, something like this;

![circle4285](https://cloud.githubusercontent.com/assets/7544231/9739333/d2350c86-5614-11e5-8808-c7b5098175b8.png)

**Comment by paulolieuthier** [comment]
Indeed.

**Comment by jleclanche** [comment]
Add a dropdown below to select which monitor the notifications should go on, too.

**Comment by stefonarch** [comment]
Closing,  still no text but not necessary.
https://github.com/lxde/lxqt-notificationd/pull/67
